### DISCUSSION TITLE
how come the tooltip on a disabled button doesn't work?

### DISCUSSION THREAD

**Discussion by lucapu88** [bootstrap-vue/bootstrap-vue#6204] [Q&A] [answered]
https://bootstrap-vue.org/docs/components/tooltip#tooltips

I'm trying it on a disabled button but it doesn't work..........

**Comment by Hiws** [comment, marked as answer]
Please check out this section of the docs. It should explain why, and how you can work around it.
<https://bootstrap-vue.org/docs/components/tooltip#disabled-elements>

**Comment by lucapu88** [comment]
@jackmu95 Can you confirm what this guy said?

**Comment by jacobmllr95** [comment]
@lucapu88 Yes it is. The docs @Hiws linked you too does exactly what you asked for.
